main.py

```diff
@@ -16,7 +16,7 @@
 from presentation.matrix_view import make_mesh_test_matrix_layout
 
 FORMAT = "[%(asctime)-15s] [%(process)d] [%(levelname)s]  %(message)s"
-logging.basicConfig(level=logging.INFO, format=FORMAT)
+logging.basicConfig(level=logging.DEBUG, format=FORMAT)
```

Comment:
As per comment in earlier PR, logging level should stay INFO

Reply:
Log level also needs to be configurable via config file and ideally also an environment variable (which are easier to pass to Docker containers as a transient config).